scripts/README.md

```diff
@@ -0,0 +1,52 @@
+# Setup & Build scripts
+
+This directory contains a few scripts to simplify setting up the Windows agent
+of webpagetest.
+
+The scripts have been developed and tested on Windows 8.1 Enterprise
```

Comment:
So I tested and it doesn't work. The script uses Get-FileHash to compute the md5 hashes, and this is only available starting with Powershell v.4.0. I've updated the script to check for this requirement and exit. I think for now we can live with this. If it turns out that we really want to compile on Windows 2012 then we can replace Get-FileHash with something else.